extension/parquet/parquet_reader.cpp

```diff
@@ -86,7 +86,7 @@ static shared_ptr<ParquetFileMetadataCache> LoadMetadata(Allocator &allocator, F
 	return make_shared<ParquetFileMetadataCache>(move(metadata), current_time);
 }
 
-static LogicalType DeriveLogicalType(const SchemaElement &s_ele) {
+static LogicalType DeriveLogicalType(const SchemaElement &s_ele, bool binary_as_string) {
```

Comment:
Can this become a method of the `ParquetReader` as well, to avoid the additional parameter?